src/ScheduleFunctions.cpp

```diff
@@ -129,22 +137,19 @@ Stmt build_loop_nest(Stmt body,
     // Put the desired loop nest into the containers vector.
     for (int i = (int)stage_s.dims().size() - 1; i >= 0; i--) {
         const Dim &dim = stage_s.dims()[i];
-        Container c = {Container::For, i, prefix + dim.var, Expr()};
-        nest.push_back(c);
+        nest.emplace_back(Container::For, i, prefix + dim.var, Expr());
     }
 
     vector<Container> pred_container;
     // Strip off the lets/ifs into the containers vector.
     while (!stmt.same_as(body)) {
-        const LetStmt *let = stmt.as<LetStmt>();
-        const IfThenElse *if_else = stmt.as<IfThenElse>();
+        const auto *let = stmt.as<LetStmt>();
```

Comment:
I find not duplicating the type easier to read. This is obviously subjective, so I would be open to following any convention the maintainers establish.

Reply:
That's the convention - we use it when the type is obvious. We don't typically use it in this particular situation, but we probably should, because it is indeed blindingly obvious here.

Reply:
Okay, then I will not change this in my final formatting commit.